tt_metal/hw/ckernels/grayskull/metal/llk_api/llk_sfpu/ckernel_sfpu_signbit.h

```diff
@@ -21,9 +21,9 @@ inline void calculate_signbit()
     {
         vFloat val = dst_reg[0];
 
-        v_if (val <= -0.0f) {
+        v_if (val < 0.0f) {
             val = 1.0f;
-        } v_elseif (val >= 0.0f) {
+        } v_else{
```

Comment:
Minor formatting but could you add a space between v_else and { for consistency?